.github/workflows/amplify_integration_tests.yaml

```diff
@@ -52,11 +56,11 @@ jobs:
 
       - name: Run Android integration tests
         uses: reactivecircus/android-emulator-runner@e790971012b979513b4e2fe70d4079bc0ca8a1ae # 2.24.0
-        timeout-minutes: 25
+        timeout-minutes: 45
         with:
           # API levels 30+ too slow https://github.com/ReactiveCircus/android-emulator-runner/issues/222
           api-level: 29
-          script: melos exec -c 1 --scope ${{ matrix.scope }} -- "deviceId=emulator-5554 retries=1 \$MELOS_ROOT_PATH/build-support/integ_test_android.sh"
+          script: melos exec -c 1 --scope ${{ matrix.scope }} -- "deviceId=emulator-5554 retries=1 small=true \$MELOS_ROOT_PATH/build-support/integ_test_android.sh"
```

Comment:
Could we make use of [tags](https://pub.dev/packages/test#tagging-tests) instead of the small/large flag?

Reply:
Looked into this and had some issues, will ping you offline.